extensions/wikia/UserRenameTool/UserRenameTool.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * A Special Page extension to rename users, runnable by staff members.
+ * Based on the RenameUser MediaWiki extension by Arnfjörð Bjarmason and Aaron Schulz.
+ *
+ * REQUIRES THE FOLLOWING EXTENSIONS TO BE ENABLED:
+ * StaffLog, TaskManager, Phalanx, SpecialPhalanx
+ */
+
+// check for dependencies
+if ( !defined( 'MEDIAWIKI' ) ) die( "This is MediaWiki extension and cannot be used standalone." );
+
+$wgExtensionCredits['specialpage'][] = array(
+	'path' => __FILE__,
+	'name' => 'UserRenameTool',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "UserRenameTool" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)